### PR TITLE
Update collection ids for collections that were moved from mundi to services.sentinel-hub.com

### PR DESCRIPTION
**Pull request by Dorothyrono**
- these collections were moved to a new bucket in aws , there they are updated regularly

### CODE REVIEW

**Review by dthiex — CHANGES_REQUESTED** [review]

File: collections/s5p-no2-tropno-daily-check.yaml [hunk 1]
```diff
@@ -21,11 +21,11 @@ Tags:
...
 Resources:
  - Group: Sentinel Hub Resources
- - EndPoint: https://shservices.mundiwebservices.com
+ - EndPoint: https://services.sentinel-hub.com
    Name: Sentinel Hub
    Role: processor
-   Type: byoc-bb92282c-b580-4206-b843-76daffbb5b0a
-   CollectionId: bb92282c-b580-4206-b843-76daffbb5b0a
+   Type: byoc-972e67a7-2ca8-4bf6-964a-11fe772e3ac2
+   CollectionId: 972e67a7-2ca8-4bf6-964a-11fe772e3ac2
```
[the remark this answers is not in the document]
Comment (dthiex) [reply collections/s5p-no2-tropno-daily-check.yaml:28]: Not sure what went wrong with checking but I just saw that there is also more recent data here (2022-03-28) so this seemed to have worked.

File: collections/chl_water_quality_saturday.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ Image: chl_water_quality_saturday/chl_water_quality_saturday.png
 EDCBrowser: https://browser.eurodatacube.com/?zoom=7&lat=43.14975&lng=7.16013&collectionId=chl_water_quality_saturday&layerId=Chlorophyll%20Anomaly%20Map&type=sentinel-hub-edc&fromTime=2020-04-17T00%3A00%3A00.000Z&toTime=2021-04-17T00%3A00%3A00.000Z
 Resolution: 300m
 GeographicalCoverage: 0.4982046708598385, 40.498973093940286, 13.82206392288208, 45.80052375793457
-TemporalAvailability: 2020-01-04 - 2021-04-17
+TemporalAvailability: 2020-01-04 - ongoing
```
[the remark this answers is not in the document]
Comment (dthiex) [reply collections/chl_water_quality_saturday.yaml:10]: @AlessandroScremin Do you maybe know how regularly the chl_water_quality_saturday collection is updated?

[the remark this answers is not in the document]
Comment (AlessandroScremin) [reply collections/chl_water_quality_saturday.yaml:10]: @dthiex 
Sorry, for the late replay.
These collections are updated every week, the last delay was due to the migration from Mundi to aws of all the collections.

SO please keep weekly update frequency (but in some exceptions they could be delayed)

**Comment by AlessandroScremin** [comment]
From My Side they are fine

@Schpidi

**Review by AlessandroScremin — APPROVED** [review]
They are fine from my side

@Schpidi

File: collections/chl_water_quality_saturday.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (dthiex) [reply collections/chl_water_quality_saturday.yaml:10]: Great, thanks for the answer! Checked and there are now also later products (e.g. 2022-04-09) available so things seem to work correctly.

**Review by dthiex — APPROVED** [review]

**Pull request merged**